dokumentlager/domene/src/main/java/no/nav/foreldrepenger/melding/kodeverk/kodeverdi/DokumentMalType.java

```diff
@@ -67,10 +67,10 @@
     ;
     public static final String KODEVERK = "DOKUMENT_MAL_TYPE";
 
-    public static final Set<DokumentMalType> FORLENGET_BREVMALER = Set.of(DokumentMalType.FORLENGET_MEDL_DOK,
-            DokumentMalType.FORLENGET_TIDLIG_SOK,
-            DokumentMalType.FORLENGET_OPPTJENING,
-            DokumentMalType.FORLENGET_DOK);
+    public static final Set<String> FORLENGET_BREVMALER = Set.of(DokumentMalType.FORLENGET_MEDL_DOK.getKode(),
```

Comment:
Jeg ville heller returnert DokumentMalType som før, men så sørget for å sjekke mot typen og ikke kode der den brukes